### PR TITLE
Add Wrapper around Cart Variant Picker

### PR DESCRIPTION
**Pull request by cbauerman**
#### Changes proposed in this Pull Request

* Import organizational changes from #61174 to simplify that PR

#### Testing instructions

1. Boot branch or use live link as Calypso Blue.
2. Add a Jetpack product to cart.
3. Go to cart and verify that nothing has changed vs production.
4. Repeat steps 2. & 3. for a WordPress.com product.

### CODE REVIEW

File: client/my-sites/checkout/composite-checkout/components/item-variation-picker/item-variation-radio-buttons.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+				id={ productSlug + variantLabel }
+				value={ productSlug }
+				checked={ isChecked }
+				disabled={ isDisabled }
+				onChange={ () => {
+					! isDisabled && onChangeItemVariant( selectedItem.uuid, productSlug, productId );
+				} }
+				ariaLabel={ translate( 'Select a different term length' ) as string }
+				label={
+					<React.Fragment>
```
Comment (sirbrillig) [inline client/my-sites/checkout/composite-checkout/components/item-variation-picker/item-variation-radio-buttons.tsx]: Really minor but you don't have to import all of `React` to get `Fragment`, you could import just `Fragment` itself.

**Review by sirbrillig — APPROVED** [review]
This looks fine to me. Other than that one type it doesn't actually change any behavior since it's not using the new components anywhere, so 👍

**Comment by cbauerman** [comment]
> This looks fine to me. Other than that one type it doesn't actually change any behavior since it's not using the new components anywhere, so 👍

Whoops! The new `ItemVariationPicker` should definitely have replaced the original in this PR 🤦. I only imported the files from the dropdown PR, which meant the `git mv` was lost.

**Review by sirbrillig — APPROVED** [review]
Ah! That makes more sense. 

This still looks good. I tested the variant picker for both wpcom and jetpack products and both seemed to work as expected.

**Pull request merged**